Framework/Kernel/src/DllOpen.cpp

```diff
@@ -10,6 +10,8 @@
 
 #include <boost/algorithm/string/predicate.hpp>
 
+#include <boost/algorithm/string/predicate.hpp>
```

Comment:
Why include twice?

Reply:
This looks like it could be a merge error?